public/components/marketing-headers/4.html

```diff
@@ -0,0 +1,116 @@
+              fill="currentColor"
+            />
+          </svg>
+        </a>
+      </div>
+
+      <div class="md:flex md:items-center md:gap-12">
+        <nav aria-label="Global" class="hidden md:block">
+          <ul class="flex items-center gap-6 text-sm">
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> About </a>
+            </li>
+
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> Careers </a>
+            </li>
+
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> History </a>
+            </li>
+
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> Services </a>
+            </li>
+
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> Projects </a>
+            </li>
+
+            <li>
+              <a class="text-gray-500 transition hover:text-gray-500/75" href="#"> Blog </a>
+            </li>
+          </ul>
+        </nav>
+
+        <div class="">
+          <button
+            type="button"
+            title="Your dashboard"
+            class="overflow-hidden rounded-full border-2 border-teal-600 bg-transparent shadow-md"
+          >
+            <img
+              src="https://images.unsplash.com/photo-1438761681033-6461ffad8d80?q=80&w=1770&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"
+              alt="Your dashboard"
+              class="size-10 object-cover"
+            />
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  
+  <div
+    class="absolute end-0 z-10 mr-4 rtl:mr-0 rtl:ml-4 mt-2 w-56 rounded-md border border-gray-100 bg-white shadow-lg dark:border-gray-800 dark:bg-gray-900"
```

Comment:
Doesn't align with the container in the `<header>`.

Reply:
There shouldn't be `dark` classes in the base component.

Reply:
Sits quite far away from the `<header>`, maybe swap the `mt-2` for `-mt-2`.

Reply:
> Doesn't align with the container in the `<header>`.

Could you be more explicit?

Reply:
![](https://github.com/user-attachments/assets/8a345ab0-e035-46a6-809f-0971b564721e)